.ladle/config.mjs

```diff
@@ -1,5 +1,6 @@
 export default {
   base: "/assessment-components/",
   outDir: "docs",
-  defaultStory: "card--default"
+  defaultStory: "print--default",
+  mode: "preview"
```

Comment:
annoying that this is necessary for the playwright navigation to work. I tried updating ladle but that made no difference.

Reply:
Yeah that was weird, and didn't seem like a caching bug either 🤷 Maybe something that gets fixed down the line